### ISSUE TITLE
strokeWidth of LinearAxisLine is always set to 1

### ISSUE THREAD

**Issue by mkisono** [reaviz/reaviz#67]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior

I'd like to make a bar chart look cleaner by removing axis lines.
To remove the line, set strokeWidth to 0 works for LinearAxisTickLine but not LinearAxisLine.
strokeWidth of LinearAxisLine is always set to 1.

## Expected behavior

strokeWidth number should be used as a line element prop.

![スクリーンショット 2022-03-30 9 03 21](https://user-images.githubusercontent.com/888153/160800651-752be538-7602-4554-aee6-b18796cce515.png)

## Minimal reproduction of the problem with instructions

See the following code snippet.

```javascript
function MyChart() {
  return (
    <BarChart
      width={350}
      height={250}
      data={data}
      gridlines={null}
      series={<BarSeries colorScheme="hotpink" bar={<Bar gradient={null} />} />}
      xAxis={
        <LinearXAxis
          type="category"
          axisLine={<LinearAxisLine strokeWidth={0} />}
          tickSeries={
            <LinearXAxisTickSeries
              line={<LinearAxisTickLine strokeWidth={0} />}
            />
          }
        />
      }
      yAxis={
        <LinearYAxis
          type="value"
          axisLine={<LinearAxisLine strokeWidth={0} />}
          tickSeries={
            <LinearYAxisTickSeries
              line={<LinearAxisTickLine strokeWidth={0} />}
            />
          }
        />
      }
    />
  );
}
```

## What is the motivation / use case for changing the behavior?

Remove axis line from chart.

**Comment by amcdnl** [comment]
Thanks! Released in 12.2.1!

**Comment by amcdnl** [comment]
FYI - another way is to just set the element null.
